### ISSUE TITLE
K600 Analyser Kinematics

### ISSUE THREAD

**Issue by padsley** [padsley/k600analyser#165]
We should remove all of these from the analyser.

It's not a good way to do the analysis and it just leads to (my and potentially others') confusion.

EDIT to explain things more fully:

When I wrote these parts originally, I then realised that it's much quicker and easier not to do kinematics in the main analyser code. I thus didn't maintain these bits of the code at all. Others have found various problems with the kinematics stuff which I wrote and have fixed them (like my lack of return statements) but I don't know if the whole thing is working.

The tl;dr version of this is: it's much better to do the kinematics away from the main K600 analyser code, and anything which I wrote (that Luna and/or Lindsay didn't fix - and I think that they fixed Quite A Lot because I'm a lazy bastard who hates return statements in non-void functions and also all living creatures) should be treated with the suspicion and contempt that they deserve.
